src/main/java/frc/trigon/robot/subsystems/collector/placeholdercollector/PlaceholderCollectorConstants.java

```diff
@@ -0,0 +1,119 @@
+            ANGLE_MOTOR_PROFILED_SETPOINT_SIGNAL = ANGLE_MOTOR.getClosedLoopReference(),
+            COLLECTION_MOTOR_VELOCITY_SIGNAL = ENCODER.getVelocity(),
+            COLLECTION_MOTOR_CURRENT_SIGNAL = COLLECTING_MOTOR.getStatorCurrent(),
+            COLLECTION_MOTOR_VOLTAGE_SIGNAL = COLLECTING_MOTOR.getMotorVoltage();
+
+
+    static {
+        configureEncoder();
+        configureCollectingMotor();
+        configureAngleMotor();
+    }
+
+    private static void configureCollectingMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.MotorOutput.Inverted = COLLECTING_MOTOR_INVERTED;
+        config.MotorOutput.NeutralMode = COLLECTING_MOTOR_NEUTRAL_MODE;
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+        COLLECTING_MOTOR.getConfigurator().apply(config);
+    }
+
+    private static void configureAngleMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.MotorOutput.Inverted = ANGLE_MOTOR_INVERTED;
+        config.MotorOutput.NeutralMode = ANGLE_MOTOR_NEUTRAL_MODE;
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.Slot0.kP = ANGLE_P;
+        config.Slot0.kI = ANGLE_I;
+        config.Slot0.kD = ANGLE_D;
+        config.Slot0.kA = ANGLE_KA;
+        config.Slot0.kG = ANGLE_KG;
+        config.Slot0.kS = ANGLE_KS;
+        config.Slot0.kV = ANGLE_KV;
+        config.Slot0.GravityType = GravityTypeValue.Arm_Cosine;
+
+        config.Feedback.FeedbackRemoteSensorID = ENCODER_ID;
+        config.Feedback.FeedbackSensorSource = ENCODER_TYPE;
+        config.Feedback.RotorToSensorRatio = CollectorConstants.ANGLE_MOTOR_GEAR_RATIO;
+
+        config.MotionMagic.MotionMagicCruiseVelocity = MOTION_MAGIC_VELOCITY;
+        config.MotionMagic.MotionMagicAcceleration = MOTION_MAGIC_ACCELERATION;
+        config.MotionMagic.MotionMagicJerk = MOTION_MAGIC_JERK;
+
+        ANGLE_MOTOR.getConfigurator().apply(config);
+        ANGLE_MOTOR.optimizeBusUtilization();
+    }
+
+    private static void configureEncoder() {
+        final CANcoderConfiguration config = new CANcoderConfiguration();
+
+        config.MagnetSensor.AbsoluteSensorRange = ENCODER_RANGE;
+        config.MagnetSensor.SensorDirection = ENCODER_DIRECTION;
+        config.MagnetSensor.MagnetOffset = ENCODER_OFFSET;
+        ENCODER.getConfigurator().apply(config);
+
+        ANGLE_MOTOR_POSITION_SIGNAL.setUpdateFrequency(100);
+        ANGLE_MOTOR_VELOCITY_SIGNAL.setUpdateFrequency(100);
+        ANGLE_MOTOR_CURRENT_SIGNAL.setUpdateFrequency(100);
+        ANGLE_MOTOR_VOLTAGE_SIGNAL.setUpdateFrequency(100);
+        ANGLE_MOTOR_PROFILED_SETPOINT_SIGNAL.setUpdateFrequency(100);
+        COLLECTION_MOTOR_VELOCITY_SIGNAL.setUpdateFrequency(100);
+        COLLECTION_MOTOR_CURRENT_SIGNAL.setUpdateFrequency(100);
+        COLLECTION_MOTOR_VOLTAGE_SIGNAL.setUpdateFrequency(100);
```

Comment:
only update the encoder status signals here, update the other ones in their correct function.